.github/workflows/build.yml

```
@@ -62,7 +14,7 @@ jobs:
         # Allowing warnings until Xcode 13 CocoaPods vendored frameworks issue is resolved: https://github.com/CocoaPods/CocoaPods/issues/10731
       - name: Run pod lib lint 
         run: pod lib lint --allow-warnings
-  carthage_xcode_beta:
+  carthage:
     name: Carthage (Xcode 13-beta)
```

```suggestion
    name: Carthage (Xcode 13)
```
Although Xcode 13 is technically unreleased. Not&nbsp;💯 &nbsp;here.

---

I think we're going to leave the 'beta' tag here so we remember to update it to the GA in September